charts/ocis/templates/appprovider/deployment.yaml

```diff
@@ -63,15 +63,15 @@ spec:
             - name: APP_PROVIDER_WOPI_APP_ICON_URI
               value: {{ $officeSuite.iconURI | quote }}
```

Comment:
The check is missing here as well. As least the iconURI is also emptied by default

Reply:
APP_PROVIDER_WOPI_APP_ICON_URI is actually optional (see empty default in https://owncloud.dev/services/app-provider/configuration/).

This also shows the invalidity of the previous default value